bindings/pyroot/pythonizations/test/uhi_test_suite.py

```diff
@@ -0,0 +1,67 @@
+from __future__ import annotations
+
+import numpy as np
+import pytest
+import ROOT
+import uhi.testing.indexing
+from ROOT._pythonization._uhi import _temporarily_disable_add_directory
+
+
+def th1_eq(self, other):
+    if self is other:
+        return True
+    return (
+        isinstance(other, type(self))
+        and self.kind == other.kind
+        and np.array_equal(self.values(), other.values())
+        and np.array_equal(self.counts(), other.counts())
+        and np.array_equal(self.variances(), other.variances())
+        and all(a == b for a, b in zip(self.axes, other.axes))
+    )
+
+
+@pytest.fixture(autouse=True)
+def patch_th1_eq(monkeypatch):
+    """
+    Temporarily patch ROOT.TH1.__eq__ for tests
+    Since __eq__ pythonizations are not defined, the comparison
+    would fall back to object identity (`is`) and would fail
+    """
+    monkeypatch.setattr(ROOT.TH1, "__eq__", th1_eq)
+
+
+class TestIndexing1D(uhi.testing.indexing.Indexing1D[ROOT.TH1]):
+    tag = ROOT.uhi
+
+    @classmethod
+    def make_histogram(cls) -> ROOT.TH1D:
+        with _temporarily_disable_add_directory():
+            h = ROOT.TH1D("h", "h", 10, 0, 1)
+            h[...] = np.array([3.0, 0.0, 2.0, 4.0, 6.0, 8.0, 10.0, 12.0, 14.0, 16.0, 18.0, 1.0])
```

Comment:
Are the values arbitrary, or should they follow any pattern? If they can be arbitrary, what is the testing suite actually testing?